src/locales/en-US/index.scss

```diff
@@ -1,4 +1,5 @@
 @import '../base-variables';
+// @import '../base-variables-dark';
```

Comment:
Is this line needed?

Reply:
I believe this change is from when I pulled the basic dark mode in as it is not a file I actually changed. Should I go ahead and remove it or leave it in?

Reply:
That is fine then.